tests/clvmc_test.py

```diff
@@ -0,0 +1,33 @@
+from tempfile import TemporaryDirectory
+
+from clvm_tools import clvmc
+
+
+INCLUDE_CODE = "((defconstant FOO 6001))"
+MAIN_CODE = """(mod (VALUE) (include "include.clvm") (+ VALUE FOO))"""
+EXPECTED_HEX_OUTPUT = "ff10ff02ffff0182177180"
```

Comment:
I think, ideally, tests should be "correct by inspection" (since we don't have tests for tests).

I think this hex could use a comment explaining to the reader why this is what we would expect. for example, that `6001` is `0x1771`. `01` is quote.

Reply:
I think of this mostly as an "idempotency test", like many of the brun tests. The main goal is to make sure the output doesn't change (and we presume it was correct at some time). That said, the comment is a good idea.